src/components/checkbox/checkbox.spec.ts

```diff
@@ -389,39 +389,15 @@ describe('Checkbox', () => {
   });
 
   describe('Validation message slots', () => {
-    async function createFixture(template: TemplateResult) {
-      element = await fixture<IgcCheckboxComponent>(template);
-    }
-
-    it('renders value-missing slot', async () => {
-      await createFixture(html`
-        <igc-checkbox required>
-          <div slot="value-missing"></div>
-        </igc-checkbox>
-      `);
-
-      await checkValidationSlots(element, 'valueMissing');
-    });
-
-    it('renders custom-error slot', async () => {
-      await createFixture(html`
-        <igc-checkbox>
-          <div slot="custom-error"></div>
-        </igc-checkbox>
-      `);
-
-      element.setCustomValidity('invalid');
-      await checkValidationSlots(element, 'customError');
-    });
-
-    it('renders invalid slot', async () => {
-      await createFixture(html`
-        <igc-checkbox required>
-          <div slot="invalid"></div>
-        </igc-checkbox>
-      `);
-
-      await checkValidationSlots(element, 'invalid');
+    it('', async () => {
+      const testParameters: ValidationContainerTestsParams<IgcCheckboxComponent>[] =
+        [
+          { slots: ['valueMissing'], props: { required: true } }, // value-missing slot
+          { slots: ['customError'] }, // custom-error slot
+          { slots: ['invalid'], props: { required: true } }, // invalid slot
+        ];
+
+      runValidationContainerTests(new IgcCheckboxComponent(), testParameters);
```

Comment:
Eh, if the only goal is to pass the tag you could do that, but to enforce the impl you could also:
```suggestion
      runValidationContainerTests(IgcCheckboxComponent, testParameters);
```
With the `runValidationContainerTests` param as  `element: Constructor<T> & IgniteComponent` to get the same behavior.

Then again I was also okay with the old fixture approach, just reducing the repetition a bit. Any benefits of `fixture` over basic create element?